i18n/miraheze/qqq.json

```diff
@@ -37,38 +37,39 @@
 	"group-abuse-member": "{{GENDER:$1|abuse team}}",
```

Comment:
Nothing in this file needs Special:MyLanguage, like you added, but I suppose it could be fine.